nf_core/__main__.py

```diff
@@ -1748,6 +1749,82 @@ def command_subworkflows_update(
     )
 
 
+# nf-core test-dataset subcommands
+@nf_core_cli.group()
+@click.pass_context
+def test_datasets(ctx):
+    """
+    Commands to manage nf-core test datasets.
+    """
+    # ensure that ctx.obj exists and is a dict (in case `cli()` is called
+    # by means other than the `if` block below)
+    ctx.ensure_object(dict)
+
+
+# nf-core test-dataset search
+@test_datasets.command("search")
+@click.pass_context
+@click.option(
+    "-b", "--branch", required=True, type=str, help="Branch in the test-datasets repository to reduce search to"
+)
+@click.option(
+    "-g",
```

Comment:
I find it more intuitive
```suggestion
    "-p",
```